### ISSUE TITLE
Fix Title

### ISSUE THREAD

**Issue by ewmcelvaney** [CodefortheCarolinas/tax-benefit-website#31]
Since <abbr> tags are showing in the title instead of only having their content read for those who need it, the name inside of the title tags should be changed to "Child Tax Credit Tax Website" to avoid accessibility issues while not having the tags on display for all users to see.
